server/tilesource/mapniksource.py

```diff
@@ -232,6 +236,8 @@ def getBounds(self, srs=None):
             }
             if srs and srs != nativeSrs:
                 inProj = pyproj.Proj(nativeSrs)
+                if isinstance(srs, six.binary_type):
+                    srs = srs.decode('utf8')
```

Comment:
We cache the bounds based on `srs`, so this might confuse the issue to change it here.  Perhaps remove these lines and just change the following line to `outProj = pyproj.Proj(srs.decode('utf8') if isinstance(srs, six.binary_type) else srs)` (that way we don't alter `srs`).